### PR TITLE
added 2023 security roadmap items

### PR DESCRIPTION
**Pull request by yash-luna**

### CODE REVIEW

**Review by michaelx11 — APPROVED** [review]
lgtm since these fall under researching!

**Comment by yash-luna** [comment]
Thanks for catching the typo @richardhuaaa and the review @michaelx11!

Everything beyond Q1 is in _researching_ so we're not committing to specific timelines. Focus is on communicating that these features are top of mind and we plan to work on them at some point.

File: vision/roadmap.md [hunk 1]
```diff
@@ -21,6 +21,18 @@ Users will be able to send messages to group chats with 2+ participants.
 
 Teams will be able to read and send encrypted DMs and announcements using a shared team identity. The account admin will be able to delegate access to and revoke access from individual team user accounts.
 
+### Post delivery storage
+
+Users will be able to store their messaging data off-network.
+
+### Signal parity security
```
Comment (saulmc) [inline vision/roadmap.md:28]: I think this should be more concrete. What are the key security properties implied by this roadmap item?

My concern is that by stating parity is the goal, without specifying what parity actually entails, we are basically just stating that XMTP has less security than Signal.

Comment (yash-luna) [reply vision/roadmap.md:28]: Good point. @richardhuaaa could 'Signal parity' be replaced by explicitly stating the security properties? What would they be?

Comment (saulmc) [reply vision/roadmap.md:28]: Another approach might be "Implement Signal protocol"

Doesn't require calling out specific security properties or explicitly compare us unfavorably.

**Pull request merged**